apps/dolly-backend/src/main/resources/db/migration/V1.7.1__UpdateTableBestillingReplaceProdOrganisasjon.sql

```diff
@@ -0,0 +1,6 @@
+--------------------------------
+-- U P D A T E   T A B L E S  --
+--------------------------------
+
+update bestilling
+set best_kriterier = REPLACE(best_kriterier, '824771332', '947064649');
```

Comment:
Du mangler en commit. For script som inneholder insert/update/delete må det alltid være med et commit stmt.

Replacen din er potensielt farlig da den også kan gi treff på mulige substrings som fødselsummer. Tryggere er å gjøre: 
REPLACE(best_kriterier, '"824771332"', '"947064649"')

Reply:
Enda tryggere vil være å skrive:
REPLACE(best_kriterier, '"virksomhetsnummer":"824771332"', '"virksomhetsnummer":"947064649"')